tools/gen-release-notes

```diff
@@ -185,9 +185,16 @@ def changelog_entries():
 
     entries_dir_relative = os.path.join('changelogs', 'unreleased')
     entries_dir = os.path.join(source_dir, entries_dir_relative)
+
+    res = []
+
     if not os.path.exists(entries_dir):
-        raise RuntimeError('The changelog entries directory {} does not '
-                           'exist'.format(entries_dir))
+        # Since git does not store directories, empty unreleased/
+        # directory won't exist on a fresh tree, which just have been
+        # released. So, do not raise here, just warn.
+        print('The changelog entries directory {} does not '
+              'exist (or was empty originally)'.format(entries_dir))
```

Comment:
I would rather add `.gitkeep` file into the directory. It is the usual way to handle such cases.

Reply:
We write resulting release notes to stdout. It is better to write error/warnings on stderr.